tests/plotting/test_plotting.py

```diff
@@ -3919,6 +3919,20 @@ def test_color_cycler():
         pl.set_color_cycler(5)
 
 
+def test_color_cycler_true():
+    pv.global_theme.color_cycler = 'default'
```

Comment:
This test is modifies the theme globally and affects other tests. Use a fixture to reset the value after testing.

```suggestion
@pytest.fixture
def _modifies_color_cycler():
    before = pv.global_theme.color_cycler
    yield
    pv.global_theme.color_cycler = before


@pytest.mark.usefixtures('_modifies_color_cycler')
def test_color_cycler_true():
    pv.global_theme.color_cycler = 'default'
```

Reply:
or maybe better is to use a per-plotter theme?

Reply:
Huh, I thought we already had an auto fixture that autoresets the theme for every single test to avoid this

Reply:
Making a new fixture:

```py
@pytest.fixture(autouse=True)
def set_default_theme():
    """Reset the testing theme for every test."""
    pv.global_theme.load_theme(pv.plotting.themes._TestingTheme())
    yield
    pv.global_theme.load_theme(pv.plotting.themes._TestingTheme())
```